livekit-agents/livekit/agents/transcription/transcription.py

```diff
@@ -0,0 +1,87 @@
+from typing import Callable
+
+from livekit import rtc
+
+from .. import tokenize
+from .stt_forwarder import STTSegmentsForwarder
+from .tts_forwarder import TTSOptions, TTSSegmentsForwarder
+
+
+class TranscriptionManager:
+    def __init__(self, room: rtc.Room):
+        self._room = room
+
+    def forward_stt_transcription(
+        self, *, participant: rtc.Participant | str, track_id: str | None = None
+    ) -> "STTSegmentsForwarder":
+        """
+        Forward STT transcription to the users. (Useful for client-side rendering)
+        """
+        identity = participant if isinstance(participant, str) else participant.identity
+        if track_id is None:
+            track_id = self._find_micro_track_id(identity)
```

Comment:
is the idea that it defaults to the first microphone track? I see that we are always passing in track_id, is the thinking that end-users would not?

Reply:
The thinking is to make it optional, so you don't have to think about it. It'll use the microphone by default, But yeah feel free for the users to pass their own track_id